rust/psibase/src/main.rs

```diff
@@ -796,94 +859,81 @@ async fn install(
 ) -> Result<(), anyhow::Error> {
     let installed = PackageList::installed(&args.api, &mut client).await?;
     let package_registry = get_package_registry(sources, client.clone()).await?;
-    let to_install = installed.resolve_new(&package_registry, packages).await?;
-
-    let mut all_account_actions = vec![];
-    let mut all_init_actions = vec![];
-    for mut package in to_install {
-        let mut account_actions = vec![];
-        package.install_accounts(&mut account_actions, sender, key)?;
-        all_account_actions.push((account_actions, package.name().to_string()));
-        let mut actions = vec![];
-        package.install(&mut actions, sender, true)?;
-        all_init_actions.push((actions, package.name().to_string()));
-    }
+    let to_install = installed
+        .resolve_changes(&package_registry, packages)
```

Comment:
The docs for `psibase install` still say, "Packages that are already installed will not be modified."

Reply:
Fixed